cmd/commento/main.go

```diff
@@ -15,23 +16,23 @@ func main() {
 	if err != nil {
 		Die(err)
 	}
-	
+
+	// Load configuration from the environment.
```

Comment:
The godotenv comes from the Ruby world where they have the 'convention over configuration' outlook. The order of searching through the environments is specified by the ruby package but not built into godotenv yet. I'm going to contact the developer to nudge him to add the functionality.